TodoListApp/src/TodoListApp/Controllers/TodoController.cs

```diff
@@ -5,37 +5,42 @@
 using Microsoft.AspNetCore.Mvc;
 using TodoListApp.Models;
 using System.Security;
+using Microsoft.AspNetCore.Authorization;
 
 namespace TodoListApp.Controllers
 {
     public class TodoController : Controller
     {
         private static ITodoListRepository memory = new InMemoryTodoListRepository();
 
+        [Authorize]
```

Comment:
you can move this attribute to the class itself. it would make all the actions require authentication